x-pack/plugins/security_solution/public/common/lib/telemetry/events/data_quality/types.ts

```diff
@@ -23,7 +23,10 @@ export interface ReportDataQualityCheckAllCompletedParams {
   ecsVersion?: string;
   isCheckAll?: boolean;
   numberOfDocuments?: number;
+  numberOfFields?: number;
   numberOfIncompatibleFields?: number;
+  numberOfEcsFields?: number;
+  numberOfCustomFields?: number;
```

Comment:
nit: Consider

```ts
  fieldsCount?: number;
  ecsFieldsCount?: number;
  customFieldsCount?: number;
```
as an alternative to:

```ts
  numberOfFields?: number;
  numberOfEcsFields?: number;
  numberOfCustomFields?: number;
```

, because the proliferation of telemetry fields that have a `numberOf` prefix (already used by the existing telemetry) makes it difficult to scan / read / locate specific fields

Reply:
We already have `numberOfIncompatibleFields` field, I prefer to be consistent and use the same prefix. So we can scan / read / locate these fields by typing "numberOf", and all of them will be suggested. I don't see the problem.